lib/nerves_hub/application.ex

```diff
@@ -3,6 +3,8 @@ defmodule NervesHub.Application do
 
   require Logger
 
+  @env Mix.env()
```

Comment:
Is this available in release mode? I remember having to add some lines to the config in order to read it from the application env

```
config :nerves_hub, env: Mix.env()
```

and then something like
```suggestion
  @env Application.compile_env(:nerves_hub, :env)
```

Reply:
Mix is available at compile time during the release so this works. We also build a release in CI so this would have failed if it was not OK to use.

Reply:
Ah yeah you're right!
I think we were using it in the function body which ended up failing